docs/ProductSummaryImage.md

```diff
@@ -44,78 +44,78 @@
 
 - `mainImageLabel` object:
 
-| Prop name | Type | Description | Default value |
-| :--------:| :--: | :---------: | :-----------: |
-| `label` | `string` | Text value that matches the value defined in the `imageLabel` field from the admin's Catalog. Once matched, it defines which product image will be the main image displayed in the Product Summary component.  If you set a label and no match is found, the main image of the product will be shown instead. | `undefined` |
-| `labelMatchCriteria`| `enum` | Criteria to define if the image's `label` searched value should be exactly as provided or if it just needs to contain the substring anywhere in the image's `label`. Possible values are: `exact` (finds the image that matches exactly the string filled in `label` field) and `contains` (finds the first image that contains the substring filled in `label` field). | `exact` |
+|      Prop name       |   Type   |                                                                                                                                                                                                  Description                                                                                                                                                                                                  | Default value |
+|:--------------------:|:--------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------:|
+|       `label`        | `string` |                                               Text value that matches the value defined in the `imageLabel` field from the Admin Catalog. Once matched, it determines which product image will be displayed as the main image in the Product Summary component.  If you set a label and no match is found, the main image of the product will be shown instead.                                               |  `undefined`  |
+| `labelMatchCriteria` |  `enum`  | Criteria for searching the image `label` value. It determines whether the image label should match exactly as provided or if it just needs to contain the substring anywhere in the image `label`. Possible values are: `exact` (finds the image that matches exactly the string filled in the `label` field) and `contains` (finds the first image that includes the substring filled in the `label` field). |    `exact`    |
```

Comment:
```suggestion
| Prop name | Type | Description | Default value |
| - | - | - | - |
| `label` | `string` | Text value that matches the value defined in the `imageLabel` field from the Admin Catalog. Once matched, it determines which product image will be displayed as the main image in the Product Summary component.  If you set a label and no match is found, the main image of the product will be shown instead. | `undefined` |
| `labelMatchCriteria` | `enum` | Criteria for searching the image `label` value. It determines whether the image label should match exactly as provided or if it just needs to contain the substring anywhere in the image `label`. Possible values are: `exact` (finds the image that matches exactly the string filled in the `label` field) and `contains` (finds the first image that includes the substring filled in the `label` field). | `exact` |
```

Reply:
@GeorgeLimaDev please do not edit original table formatting, especially if it's already in compact markdown. It's much easier for tech writers to edit and review compact markdown tables without many blank spaces.